src/uu/sort/src/sort.rs

```diff
@@ -276,7 +358,58 @@ With no FILE, or when FILE is -, read standard input.",
     exec(files, &settings)
 }
 
+fn exec_head(files: Vec<String>, settings: &Settings) -> i32 {
```

Comment:
I know the main function is called "exec" but head is probably enough here as function name